Apps/Medication/src/Models/Prescription.cs

```diff
@@ -0,0 +1,93 @@
+﻿//-------------------------------------------------------------------------
+// Copyright © 2019 Province of British Columbia
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//-------------------------------------------------------------------------
+namespace HealthGateway.MedicationService.Models
+{
+    /// <summary>
+    /// The patient data model.
+    /// </summary>
+    public class Prescription
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Prescription"/> class.
+        /// </summary>
+        public Prescription()
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Prescription"/> class.
+        /// </summary>
+        /// <param name="phn">The patient's personal health number to scope the medication statement.</param>
+        public Prescription(string phn)
```

Comment:
The phn is not being used. Do we need this constructor?

Reply:
Opened AB#7473 to discuss this.